3rdparty/libconfig/libconfig.c

```diff
@@ -195,6 +215,17 @@ static void __config_write_value(const struct config_t *config,
           fprintf(stream, "0x%X", (unsigned int)(value->ival));
           break;
 
+        case CONFIG_FORMAT_BIN:
+        {
+          char buffer[33]; // 32 bits + null
```

Comment:
```suggestion
          char buffer[33]; // 32 bytes + null
```

Reply:
what I meant here is that the binary literal can be up to 32 bits long (`0b0101010101...` up to 32 digits), so we have a buffer that can hold 32 of these individual bits represented as bytes plus another for null termination.. although it might be less confusing to just write `32 bytes + null`